tests/test_tenant_unauthorized.py

```diff
@@ -0,0 +1,155 @@
+from unittest.mock import MagicMock
+import pytest
+from assertpy import assert_that
+from dataall.base.api import bootstrap
+from dataall.base.loader import load_modules, ImportMode
+from dataall.base.context import RequestContext
+from dataall.base.db.exceptions import TenantUnauthorized
+import inspect
+
+
+load_modules(modes={ImportMode.API})
+
+NO_CHECK_PERMS = [
+    'Mutation.updateGroupTenantPermissions',
+    'Mutation.updateSSMParameter',
+    'Mutation.createNetwork',
+    'Mutation.deleteNetwork',
```

Comment:
is this to say Operations like `createNetwork` and `deleteNetwork` do not have `MANAGE_XXXX` checks on them? They do have `MANAGE_ENVIRONMENTS` ... and I think a handful of tohers on this list are simialr 

Or are these mutations that would need some additional work on the backend app logic side to conform to this pattern?